src/utils/customHTML.ts

```diff
@@ -0,0 +1,109 @@
+import * as cheerio from "cheerio"
+import type { Element } from "cheerio"
+
+/**
+ * Custom HTML tags supported by our CMS.
+ * Note: Custom HTML tags should contain a "-" in their name.
+ */
+export const CUSTOM_HTML_TAGS: string[] = ["instagram-post"]
+
+/**
+ * Custom HTML attributes supported by our CMS.
+ */
+export const CUSTOM_HTML_ATTRIBUTES: string[] = [
+  "post-id", // custom attribute for instagram-post tag
+  "captioned", // custom attribute for instagram-post tag
+]
+
+/**
+ * Retrieve the HTML for embedding an Instagram post
+ * @param postId the ID of the Instagram post
+ * @param isCaptioned whether to include the post caption
+ * @returns the Instagram post HTML embed code
+ */
+export const getInstagramEmbedCode = (
+  postId: string | undefined,
```

Comment:
why are we allowing an empty `postId` here? i would think that the `postId` should exist in order for the embed to exist